ivy/functional/backends/numpy/experimental/statistical.py

```diff
@@ -5,6 +5,24 @@
 from ivy.func_wrapper import with_supported_dtypes
 from . import backend_version
 
+def histogramdd(
+        input: np.ndarray,
+        /,
+        *,
+        bins: Optional[Union[Tuple[int],int,Tuple[float]]] = 10,
```

Comment:
The type hinting for generic arguments that do not involve any native elements should be the same across each file